pages/common/chmod.md

```diff
@@ -30,3 +30,7 @@
 - Change permissions recursively giving [g]roup and [o]thers the ability to [w]rite:
 
 `chmod -R g+w,o+w {{directory}}`
+
+- Recursively give [a]ll users [r]ead permissions to all contents in a directory. Makes subdirectories e[X]ecutable but not files:
```

Comment:
What do you think about this?
```suggestion
- Recursively give [a]ll users [r]ead and e[X]ecute permissions to a directory and its sub-directories, but not to the files it contains:
```

Reply:
It's better, but makes it sound like that only directories get read access, when files also get read access. Is this more clear?
```suggestion
- Recursively give [a]ll users [r]ead permissions to files and e[X]ecute permissions to sub-directories within a directory:
```

Reply:
Perfect, I like it.